i18n/de.json

```diff
@@ -2356,10 +2388,34 @@
     "translation": "Die Berechtigungen für die Erneuerung des Webhooks-Tokens sind ungeeignet"
   },
   {
+    "id": "api.webrtc.init.debug",
+    "translation": "Initialisiere WebRTC-API-Routen"
+  },
+  {
+    "id": "api.webrtc.not_available.app_error",
+    "translation": "WebRTC ist auf diesem Server nicht verfügbar."
+  },
+  {
     "id": "api.websocket_handler.invalid_param.app_error",
     "translation": "Ungültiger {{.Name}} Parameter"
   },
   {
+    "id": "authentication.permissions.team_invite_user.description",
+    "translation": "Die Möglichkeit Benutzer in ein Team einzuladen"
+  },
+  {
+    "id": "authentication.permissions.team_invite_user.name",
+    "translation": "Benutzer einladen"
+  },
+  {
+    "id": "authentication.permissions.team_use_slash_commands.description",
+    "translation": "Die Möglichkeit Slash Befehle zu verwenden"
+  },
+  {
+    "id": "authentication.permissions.team_use_slash_commands.name",
+    "translation": "Slash Befehle verwenden"
+  },
+  {
     "id": "cli.license.critical",
     "translation": "Feature erfordert Enterprise Lizenz. Bitte kontaktieren Sie Ihren Systemadministrator wegen Upgrade Ihrer Enterprise Lizenz."
```

Comment:
Thanks @monsdar sadly any changes to the translations have to be done in the Translation server, that way all the changes can be included in a future PR.